.github/workflows/main.yml

```diff
@@ -90,6 +90,21 @@ jobs:
       - name: Test
         run: yarn && yarn test
 
+      - name: Coveralls
+        run: yarn report && yarn coveralls
+        env:
+          GITHUB_TOKEN: ${{ secrets.github_token }}
+          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+          COVERALLS_PARALLEL: true
+          COVERALLS_SERVICE_NUMBER: ${{ github.run_id }}
+
+  finish:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+    - name: Coveralls Finished
```

Comment:
Does it publish it for all five versions of Node?

Reply:
Yes, it runs a job for each of the 5 versions and publishes them all in one build. Here's an example: https://coveralls.io/builds/51891392

I'm going to rename the jobs more clearly so they have the Node versions in them. Thanks for pointing this out!

Reply:
Updated example: https://coveralls.io/builds/51892051